src/Model/AdminStats.php

```diff
@@ -229,6 +229,27 @@ private function groupStatsByUsername(array $data): array
         return $users;
     }
 
+    /**
+     * Get the "totals" row.
+     * @return array containing as keys the counts.
+     */
+    public function getTotalsRow(): array
+    {
+        $totalsRow = [];
+        foreach ($this->adminStats as $object) {
+            foreach ($object as $key => $value) {
+                if ('username' !== $key && 'user-groups' !== $key) {
+                    if (isset($totalsRow[$key])) {
+                        $totalsRow[$key] += $value;
+                    } else {
+                        $totalsRow[$key] = $value;
+                    }
+                }
+            }
```

Comment:
The full nitpick:

```suggestion
        foreach ($this->adminStats as $data) {
            foreach ($data as $action => $count) {
                if ('username' === $action || 'user-groups' === $action) {
                    continue;
                }
                $totalsRow[$action] = ($totalsRow[$action] ?? 0) + $count;
            }
        }
```

or use a null coalescence assignment to set the counter:

```
$totalsRow[$action] ??= 0;
$totalsRow[$action] += $count;
```

Reply:
Also, it seems like it should be straightforward to add a test case for this method to `AdminStatsTest`, should you feel compelled :)

Reply:
BTW, on null coalescing: IIRC there's quite a lot of the older code that goes the verbose "if not there then 1 else += 1" way.

Added a test. (The issue with statistics is that it's often barely testable. LIke, 5k values? So in the end most of the time I don't even check.)